sidecar-injector/bin/build

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+#
+# Build CyberArk sidecar injector mutating webhook service
+# usage: ./bin/build
+set -ex
+
+if [ -z "$1" ]; then echo "tag not provided"; exit 1; fi
```

Comment:
Blocks like this one should be split-lined/indented for easier visual readability